Tests/TuistLoaderTests/Up/UpMintTests.swift

```diff
@@ -0,0 +1,158 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistGenerator
+import TuistSupport
+import XCTest
+
+@testable import TuistLoader
+@testable import TuistSupportTesting
+
+final class UpMintTests: TuistUnitTestCase {
+    var upHomebrew: MockUp!
+
+    override func setUp() {
+        super.setUp()
+        upHomebrew = MockUp()
+    }
+
+    override func tearDown() {
+        upHomebrew = nil
+        super.tearDown()
+    }
+
+    func test_init() throws {
+        let temporaryPath = try self.temporaryPath()
+        let json = JSON(["linkPackagesGlobally": JSON.bool(true)])
+        let got = try UpMint(dictionary: json, projectPath: temporaryPath)
+        XCTAssertTrue(got.linkPackagesGlobally)
+    }
+
+    func test_isMet_when_homebrew_is_not_met() throws {
+        let subject = UpMint(linkPackagesGlobally: false, upHomebrew: upHomebrew)
+        let temporaryPath = try self.temporaryPath()
+
+        upHomebrew.isMetStub = { _ in false }
+
+        XCTAssertFalse(try subject.isMet(projectPath: temporaryPath))
+    }
+
+    func test_isMet_when_mintfile_doesnt_exist() throws {
+        let subject = UpMint(linkPackagesGlobally: false, upHomebrew: upHomebrew)
+        let temporaryPath = try self.temporaryPath()
+
+        upHomebrew.isMetStub = { _ in true }
+
+        XCTAssertThrowsError(try subject.isMet(projectPath: temporaryPath)) { error in
```

Comment:
Since `UpMintError` conforms `Equatable`, you can simplify this assertion leveraging this [helper](XCTAssertThrowsSpecific)